src/repos/controllers/package.py

```diff
@@ -101,16 +101,38 @@ def info(self, name):
         version = self.field("version")
         return repos.Artifact._info(name = name, version = version)
 
-    @appier.route("/packages/<str:name>/artifacts", "GET", json = True)
+    @appier.route("/packages/<str:name>/artifacts", "GET", json=True)
     def artifacts(self, name):
         self.ensure_auth()
-        object = appier.get_object(alias = True, find = True)
+
+        # Perform the find query
+        object_ = appier.get_object(alias=True, find=True)
         artifacts = repos.Artifact.find(
-            package = name,
-            map = True,
-            sort = [("timestamp", -1)],
-            **object
+            package=name,
+            map=True,
+            sort=[("timestamp", -1)],
+            **object_

```

Comment:
I think we could use this opportunity to enable the `sort` parameter to work. As is, it is not possible to sort by anything else on the API call. I mean, adding the default `[("timestamp", -1)]` to the `object` only if the sorting was not specified already.

Reply:
Good idea. Will cover this in a separate pull request 👍